Sources/ClientRuntime/Networking/Http/HttpTelemetry.swift

```diff
@@ -160,10 +160,30 @@ internal enum HttpMetricsAttributesKeys {
     internal static let serverAddress = AttributeKey<String>(name: "server.address")
 }
 
-internal struct HttpMetricsUsage {
+internal actor HttpMetricsUsage {
```

Comment:
`HttpMetricsUsage` is now an actor, to enforce exclusive access to its properties.

Async setter methods are added for its properties since actor properties may not be set from outside the actor.